stdlib/array.gr

```diff
@@ -392,7 +408,13 @@ export let flatMap = (fn, array) => {
  * @since v0.3.0
  */
 export let every = (fn, array) => {
-  reduce((acc, value) => { acc && fn(value) }, true, array)
```

Comment:
This didn't break how I expected it to—I know that because there's a block it broke, but I would have imagined it to break like this:
```
reduce((acc, value) => {
  acc && fn(value)
}, true, array)
```